extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js

```diff
@@ -0,0 +1,60 @@
+import {
+    billTheLizard,
```

Comment:
This can be in one line:
`import { billTheLizard, context } from '@wikia/ad-engine';`